### DISCUSSION TITLE
can't get TMC to work in Sweden with Audi Connect expired

### DISCUSSION THREAD

**Discussion by Falconer999** [Mr-MIBonk/M.I.B._More-Incredible-Bash#327] [Q&A]
Hi all, I played around for last few days to get it to work, but without any result. would anyone be able to tell me if I would need to inject my own custom file or how should I set it up in M.I.B by using the files that provided with the stock program package, please?

**Comment by Warriorr** [comment]
Sweden stopped TMC-service in August 2022 so I believe that's the issue.

https://bransch.trafikverket.se/tjanster/trafiktjanster/TMC---Traffic-Message-Channel/
